hack/quickstart/init-master.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -euo pipefail
+set -euox pipefail
```

Comment:
Do we want to leave the `-x` option, or was that during debugging? Or if it's generally useful should we turn it on for `init-node.sh` also?

Reply:
It was for debugging, I'll remove, though its a bit useful